gluten-ut/common/src/test/scala/org/apache/spark/sql/GlutenTestsTrait.scala

```diff
@@ -128,11 +128,10 @@ trait GlutenTestsTrait extends GlutenTestsCommonTrait {
     val expr = resolver.resolveTimeZones(expression)
     assert(expr.resolved)
 
-    val catalystValue = CatalystTypeConverters.convertToCatalyst(expected)
-
     if(canConvertToDataFrame(inputRow)) {
-      glutenCheckExpression(expr, catalystValue, inputRow)
+      glutenCheckExpression(expr, expected, inputRow)
     } else {
+      val catalystValue = CatalystTypeConverters.convertToCatalyst(expected)
```

Comment:
Does this mean the execution is on vanilla Spark, and the unit test is not actually verified? If so, can we add a warning here?